spacetemplate/parsed_template_blackbox_test.go

```diff
@@ -0,0 +1,267 @@
+package spacetemplate_test
+
+import (
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+	"github.com/almighty/almighty-core/spacetemplate"
+	"github.com/almighty/almighty-core/workitem"
+	"github.com/almighty/almighty-core/workitem/link"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// getValidTestTemplate returns a test template in unparsed format. See
+// getValidTestTemplateParsed() for the parsed representation of this template
+func getValidTestTemplate() string {
+	return `
+work_item_types:
+  custom_bug:
+    # planneritem "magically" resolves to 86af5178-9b41-469b-9096-57e5155c3f31 for the witrepo.Create()
+    extends_type: planneritem
+    description: "a basic work item type"
+    name: "Custom Bug"
+    icon: fa fa-bug
+    fields:
+      title:
+        label: "Title"
+        description: "The title of the bug"
+        required: yes
+        type:
+          kind: string
+      state:
+        label: "State"
+        description: "The state of the bug"
+        required: NO
+        type:
+          kind: enum
+          base_type: integer
```

Comment:
Fixed in 1766a2143f2afe6ed055f95ed3a8c0c3a834fa18 and a follow-up commit in cf9d33b20db9c82855280cd7adb32472d69843a3.